src/MediaBundleForm.php

```diff
@@ -129,21 +152,23 @@ public function form(array $form, FormStateInterface $form_state) {
       '#tree' => TRUE,
     );
 
-    foreach ($plugins as $plugin => $definition) {
-      $plugin_configuration = $bundle->getType()->getPluginId() == $plugin ? $bundle->type_configuration : array();
-      $form['type_configuration'][$plugin] = array(
-        '#type' => 'container',
-        '#states' => array(
-          'visible' => array(
-            ':input[name="type"]' => array('value' => $plugin),
-          ),
-        ),
-      );
+    $plugin = $bundle->getType()->getPluginId();
```

Comment:
It seems that something is wrong with this part:
- open bundle add form
- Generic is pre-selected
- change it to something else
- AJAX request happens
- plugin configuration part stays the same (i.e. "This type provider doesn't need configuration.")